tensorwaves/physics/helicityformalism/amplitude.py

```diff
@@ -275,9 +275,9 @@ def __init__(self, recipe: list, builder):
             )
 
     def __call__(self, dataset):
-        seq_amp = tf.complex(atfi.const(1.0), atfi.const(0.0))
+        seq_amp = atfi.complex(atfi.const(1.0), atfi.const(0.0))
```

Comment:
Why use `amplitf` here?

Reply:
Oh, I was just testing something. Ideally we want all of the calls to be amplitf calls, because there it is foreseen to have a tensorflow or numpy backend, which can be chosen dynamically. So by using the amplitf code you are flexible in chosing the backend. But all of the tf calls would have to be swapped by amplitf calls and amplitf also has to be extended.